spec/amp-var-substitutions.md

```diff
@@ -65,6 +65,15 @@ For instance:
 ```
 may make a request to something like `https://foo.com/pixel?path=%2Fpage1.html`.
 
+**CLIENT_ID**
+
+Use the special string `CLIENT_ID` to add a per AMP document source origin client identifier.
```

Comment:
Need to document the one required and one optional argument.

Lets say
"…to add a per document-source-origin (The origin of the website where you publish your AMP doc) and user identifier. The CLIENT_ID will be the same for the same user if they visit again within one year. It should behave roughly similar to a cookie storing a session id for one year."